capitalone/rubicon-ml
# deprecate async submodules

## What
  * removing the async module (for now - we can always bring it back later if necessary)
    * there's a lot of duplicated code that hasn't been kept up to date with the parts of the library its duplicated from
      * if people were actually using the async stuff they would've been complaining about how it isn't working
    * there are probably better ways to handle async read/write with `fsspec` directly rather than implementing it ourselves

## How to Test
  * validate the tests pass: `python -m pytest`
  * build the docs and make sure there are no more references to the async stuff: `make clean html open` from rubicon-ml/docs
